### PR TITLE
Add MagTag Weather

### PR DESCRIPTION
**Pull request by caternuson**
Code and assets for MagTag weather display.

### CODE REVIEW

File: MagTag_Weather/magtag_weather.py [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    "October",
+    "November",
+    "December",
+)
+magtag = MagTag()
+
+# ----------------------------
+# Backgrounnd bitmap
+# ----------------------------
+bg_bmp, bg_pal = adafruit_imageload.load(BACKGROUND_BMP)
```
Comment (ladyada) [inline MagTag_Weather/magtag_weather.py:40]: magtag.set_background doesnt work here?

**Review by ladyada — APPROVED** [review]

File: MagTag_Weather/magtag_weather.py (hunk 1 above)
Comment (caternuson) [reply MagTag_Weather/magtag_weather.py:40]: Let me look. Mainly took that approach for more z ordering control of background along with other items.

Comment (ladyada) [reply MagTag_Weather/magtag_weather.py:40]: no biggie, i think we sorta overrode the magtag graphics system here, which is ok!

Comment (caternuson) [reply MagTag_Weather/magtag_weather.py:40]: Ok, that'll work here. Left over thing from tides, where "background" was more of an intermediate layer thing.

**Pull request merged**